### ISSUE TITLE
KeyError: 'O' makefile:7: recipe for target 'run' failed

### ISSUE THREAD

**Issue by oabid15** [guillaumegenthial/sequence_tagging#88]
I have prepared my test file containing this 3 lines : 

Survey B-DOC
Report I-DOC
Geographic I-DOC

when I make run, I get this error : 

default = tags[NONE]
KeyError: 'O'
makefile:7: recipe for target 'run' failed
make: *** [run] Error 1

Could you please help me.

**Comment by RishabhJain-Github** [comment]
Hey did you got the answer to this I am facing a similar problem!

**Comment by RishabhJain-Github** [comment]
Hey you have to preprocess the data as mentioned in the  test.txt, file that solved the problem for me.

**Comment by RishabhJain-Github** [comment]
Yeah because the model gives default tag as 'O', therefore while training at least one word should have the 'O" tag. Hope this helps.
